README.md

```diff
@@ -1,33 +1,31 @@
-# Operate First template for repositories
+# Project Meteor
 
-Derive new repositories from this template
+Similar to how a meteoroid entering the atmosphere, becomes visible as a meteor and finally hitting the Earth's surface as a meteorite, we aim to create better visibility and impact of our team's work through project *Meteor*.
 
-List of featurese:
+It is an AICoE combined team effort to provide a product for Data Scientists and other users alike where they can interact and leverage our services, tools and technology for developing various applications. The product can also help in introducing, sharing and promoting our team's capabilities with others.
```

Comment:
are you able to give some context on AICoE for non-AICoE-ers? :)

Reply:
yes good idea, ill include a sentence for this

Reply:
```suggestion
Project Meteor is a combined effort across the AI CoE to provide a single tool for data scientists and other users where they can interact with, explore and leverage all of our services, tools and technologies for developing intelligent applications. 
```